app/models/project_account.rb

```diff
@@ -0,0 +1,11 @@
+class ProjectAccount < ActiveRecord::Base
+  belongs_to :project
+  belongs_to :bank
+  belongs_to :user
+
+  delegate   :display_bank_account, :display_bank_account_owner, :display_address, to: :decorator
```

Comment:
you can remove this ;)
just use record.decorate.method_name on view, if we use delegate we need to delegate every new method on decorators.